tests/queries/0_stateless/01557_field_infinite_convert_to_number.sql

```diff
@@ -1 +1 @@
-SET max_threads = nan; -- { serverError 70 }
+SET max_threads = nan; -- { serverError 69 }
```

Comment:
70 is CANNOT_CONVERT_TYPE, 69 is ARGUMENT_OUT_OF_BOUND - should it be 70 still? also, we can put string here:
```suggestion
SET max_threads = nan; -- { serverError CANNOT_CONVERT_TYPE }
```